src/Inventory.java

```diff
@@ -0,0 +1,280 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/**
+ * Created by melmo on 12/1/16.
+ */
+public class Inventory {
+    private int CURRENT_ITEM_ID = 1000;
```

Comment:
Do not use all caps unless it is a constant variable.  Anything that isn't `final` is camel case.